Jenkinsfile

```
@@ -488,6 +490,57 @@ pipeline {
                         }
                     }
                 }
+                stage('Build on CentOS 7 Bullseye') {
+                    when {
+                        beforeAgent true
+                        allOf {
+                            not { environment name: 'NO_CI_TESTING',
+                                  value: 'true' }
+                            expression {
+                                cachedCommitPragma(pragma: 'Skip-bullseye',
+                                                   def_val: 'true') != 'true'
```

Why does this default to `true`?

If it really should default to being skipped, you can extend (to take/override a default state of `false`) `skip()`: https://github.com/daos-stack/daos/blob/570916a6a93a1f22e60e392dde2f70222ee75b42/Jenkinsfile#L34-L36
```suggestion
                                expression { ! skip_stage('bullseye', true) }
```

You could then apply this to:
https://github.com/daos-stack/daos/blob/570916a6a93a1f22e60e392dde2f70222ee75b42/Jenkinsfile#L291-L292

---

Not quite as simple of a change, but looks like I have it working now to accept either 'true' or true as a parameter.
We may want to look at other cases of passing a boolean as a string.